clients/errors.go

```diff
@@ -8,6 +8,8 @@ type NotFoundError struct{}
 func (e *NotFoundError) Error() string { return "Item not found" }
 
 func IsNotFoundErr(err error) bool {
-	return err.Error() == "Item not found" || strings.HasPrefix(err.Error(), "no item found")
-	// return errors.Is(err, &NotFoundError{})
+	if err == nil {
```

Comment:
please add license header

Reply:
Thanks for reminding, the license header is added.